ghc.nix

```diff
@@ -270,7 +278,9 @@ hspkgs.shellFor rec {
     ${lib.optionalString withDocs "export FONTCONFIG_FILE=${fonts}"}
 
     # N.B. This overrides CC, CONFIGURE_ARGS, etc. to configure the cross-compiler.
-    ${lib.optionalString withWasiSDK "addWasiSDKHook"}
+    # FIXME: This bit is undocumented so I don't know what effect renaming it to
```

Comment:
It's defined at https://gitlab.haskell.org/ghc/ghc-wasm-meta/-/blob/master/pkgs/wasi-sdk-setup-hook.sh and sets up various env vars to override the existing toolchain provided by `stdenv`. So don't change the name, but indeed it might be worth adding a comment that refers to this url for clarity